core/mutator/entry/mutation.go

```diff
@@ -148,3 +153,19 @@ func (m *Mutation) sign(signers []signatures.Signer) (*pb.Entry, error) {
 	m.entry.Signatures = sigs
 	return m.entry, nil
 }
+
+// Check verifies that an update was successfully applied.
+// Returns nil if newLeaf is equal to the entry in this mutation.
+func (m *Mutation) Check(newLeaf []byte) error {
+	// XXX Mutations are no longer stable serialized byte slices, so we need to use
```

Comment:
XXX - is this a todo?

Reply:
It's more of a warning and a TODO that's related to our discussion of ObjectHash. 
Storing objects is really nice because I get to avoid lots of `UnMarshals`, but then I need to do object comparison in a reliable way...